internal/k8s_watcher/k8s_client/client.go

```diff
@@ -1,4 +1,4 @@
-package k8s_client
+package k8sclient
```

Comment:
No need to remove underscore because rule has been excluded, I think

Reply:
Removed